x-pack/plugins/aiops/public/components/change_point_detection/fields_config.tsx

```diff
@@ -138,25 +158,244 @@ export interface FieldPanelProps {
+          ? i18n.translate('xpack.aiops.changePointDetection.attachSelectedChartsLabel', {
+              defaultMessage: 'Attach selected charts',
+            })
+          : i18n.translate('xpack.aiops.changePointDetection.attachChartsLabel', {
+              defaultMessage: 'Attach charts',
+            }),
+      items: [
+        {
+          name: i18n.translate('xpack.aiops.changePointDetection.attachToDashboardLabel', {
+            defaultMessage: 'To dashboard',
+          }),
+          panel: 'attachToDashboardPanel',
+        },
+        {
+          name: i18n.translate('xpack.aiops.changePointDetection.attachToCaseLabel', {
+            defaultMessage: 'To case',
+          }),
+          disabled: caseAttachmentButtonDisabled,
+          ...(caseAttachmentButtonDisabled
+            ? {
+                toolTipPosition: 'left',
+                toolTipContent: i18n.translate(
+                  'xpack.aiops.changePointDetection.attachToCaseTooltipContent',
+                  {
+                    defaultMessage: 'Select change points to attach',
+                  }
+                ),
+              }
+            : {}),
+          onClick: () => {
+            openCasesModalCallback({
+              timeRange,
+              fn: fieldConfig.fn,
+              metricField: fieldConfig.metricField,
+              dataViewId: dataView.id,
+              ...(fieldConfig.splitField
+                ? {
+                    splitField: fieldConfig.splitField,
+                    partitions: selectedPartitions,
+                  }
+                : {}),
+            });
+          },
+        },
+      ],
+    },
+    {
+      id: 'attachToDashboardPanel',
+      title: i18n.translate('xpack.aiops.changePointDetection.attachToDashboardTitle', {
+        defaultMessage: 'Attach to dashboard',
+      }),
+      size: 's',
+      content: (
+        <EuiPanel paddingSize={'s'}>
+          <EuiSpacer size={'s'} />
+          <EuiForm>
+            <EuiFormRow fullWidth>
+              <EuiSwitch
+                label={i18n.translate('xpack.aiops.changePointDetection.applyTimeRangeLabel', {
+                  defaultMessage: 'Apply time range',
+                })}
+                checked={dashboardAttachment.applyTimeRange}
+                onChange={(e) =>
+                  setDashboardAttachment((prevState) => {
+                    return {
+                      ...prevState,
+                      applyTimeRange: e.target.checked,
+                    };
+                  })
+                }
+                compressed
+              />
+            </EuiFormRow>
+            {isDefined(fieldConfig.splitField) && selectedPartitions.length === 0 ? (
+              <EuiFormRow
+                fullWidth
+                label={
+                  <FormattedMessage
+                    id="xpack.aiops.changePointDetection.maxSeriesToPlotLabel"
+                    defaultMessage="Max series"
```

Comment:
It might be worth adding a tooltip briefly explaining what "Max series" means.

Reply:
+1 to extra clarification here. For the anomaly charts embeddable we use this as the label `Maximum number of series to plot`, which could probably be used here too.

Reply:
hey @szabosteve, yes indeed, I think it's worth it, I'm open to suggestions 🙂

Reply:
@darnautov I suggested a possible tooltip below. Please have a look and let me know what you think or if I misunderstood the function of `Max series`. 

```
The maximum number of change points to visualize.
```

Reply:
Added in https://github.com/elastic/kibana/pull/162796/commits/f4cd51665a813ac7b95c9bcf3b2525fc8b5ea87f

<img width="361" alt="image" src="https://github.com/elastic/kibana/assets/5236598/d620e0c0-d33d-4d42-a874-c3f2a35f0d8d">